demosplan/DemosPlanCoreBundle/Tools/DocxImporterHttp.php

```diff
@@ -53,6 +53,7 @@ public function importDocx(File $file, string $elementId, string $procedure, str
             ];
         } catch (Exception $e) {
             $this->logger->error('Error while creating docx with http: '.$e->getMessage());
+            $this->logger->error('Response body: '. $response->getContent(false));
```

Comment:
It is very possible that `$response` is undefined here, which would trigger an error while trying to log.Maybe wrap it in a check to only log when present.

Reply:
good catch, thx f25b5bda